qdrant_client/qdrant_remote.py

```diff
@@ -1075,7 +1075,7 @@ def update_vectors(
                 grpc.UpdatePointVectors(
                     collection_name=collection_name,
                     wait=wait,
-                    vectors=vectors,
```

Comment:
Our Python client now still uses `vectors`, while our gRPC/REST API uses `points`.

Our documentation also suggests to use `points`: https://qdrant.tech/documentation/concepts/points/#update-vectors

I prefer `points` because you provide a list of points (with vectors), and not a list of vectors directly.

Do we leave as is and update our documentation, or do we switch to `points`? I like the latter better.

Reply:
let's switch